### PR TITLE
confirm modals for deploy, redeploy, and rollback

### PR DESCRIPTION
**Pull request by GraysonNull**

### CODE REVIEW

File: kotsadm/web/src/components/apps/AppVersionHistory.jsx [hunk 1]
```diff
@@ -314,28 +317,66 @@ class AppVersionHistory extends Component {
...
+      this.setState({
+        displayConfirmDeploymentModal: true,
+        confirmIsRedeploy: false,
+        confirmIsRollback: true,
+        versionToDeploy: version,
+      });
+    } else {
+      this.setState({
+        displayConfirmDeploymentModal: true,
+        confirmIsRedeploy: true,
```
Comment (salahalsaleh) [inline kotsadm/web/src/components/apps/AppVersionHistory.jsx:364]: how about also setting `confirmIsRollback` to `false` here? you could also use string type and merge those into 1 variable

File: kotsadm/web/src/components/apps/AppVersionHistoryRow.jsx [hunk 2]
```diff
@@ -82,7 +82,7 @@ function renderVersionAction(version, nothingToCommitDiff, app, history, deployV
         <button
           className={classNames("btn", { "secondary dark": isRollback, "secondary blue": isSecondaryBtn, "primary blue": isPrimaryButton })}
           disabled={version.status === "deploying"}
-          onClick={() => needsConfiguration ? history.push(`/app/${app.slug}/config/${version.sequence}`) : deployVersion(version)}
+          onClick={() => needsConfiguration ? history.push(`/app/${app.slug}/config/${version.sequence}`) : isRollback ? deployVersion(version, true) : deployVersion(version)}
```
Comment (salahalsaleh) [inline kotsadm/web/src/components/apps/AppVersionHistoryRow.jsx:85]:
```suggestion
          onClick={() => needsConfiguration ? history.push(`/app/${app.slug}/config/${version.sequence}`) : deployVersion(version, isRollback)}
```

Comment (GraysonNull) [reply kotsadm/web/src/components/apps/AppVersionHistoryRow.jsx:85]: I'm not sure it's this simple. the `deployVersion` function is passed to it and can be different. I only want to pass that var if it is rollback

Comment (salahalsaleh) [reply kotsadm/web/src/components/apps/AppVersionHistoryRow.jsx:85]: ah i see.

**Review by salahalsaleh — APPROVED** [review]

**Pull request merged**